### ISSUE TITLE
Add VIP label

### ISSUE THREAD

**Issue by chrisrohn** [mxew/firetable-user#150]
/vip label to identify Very Important Persons

**Comment by ghost** [comment]
Part of #140?

**Comment by chrisrohn** [comment]
Yeah I think this is for situations where artists pop in and we can tag
them with a VIP command: /vip Username
